docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
-  Selects the 1st person in the results of the `find` command.
+* `find cs2103`<br>
+  Returns any task or event pertaining to cs2103
+
+### 3.7. Deleting a task/event without moving it to the expired list: `delete`
+
+> Just like edit, user can delete tasks and events in the active list by using their indexes specified in the most recent list. This option can also be used for tasks or events already in the expired list from which the user wants to get rid of.
+
+Format: `delete INDEX`
 
-### 2.8. Clearing all entries : `clear`
+### 3.8. Clearing all expired tasks and events : `clear`
 
-Clears all entries from the address book.<br>
 Format: `clear`
 
-### 2.9. Exiting the program : `exit`
+### 3.9 Viewing other material: `view`
 
-Exits the program.<br>
-Format: `exit`
+**View command history**: 
 
-### 2.10. Saving the data
+Format: `view history`
 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+**View the calendar**: 
 
-## 3. FAQ
+Format: `view calendar`
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with
-       the file that contains the data of your previous Address Book folder.
+**View free time during the day/week/month** : 
+
+Format: `view free day | week | month`
+
+### 3.10 Exiting the program : `exit`
+
+Format: `exit`
+
+### 3.11 Undoing the last command: `undo`
+
+Format: `undo`
 
-## 4. Command Summary
+### 3.12 Recycle 
 
-* **Add**  `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` <br>
-  e.g. `add James Ho p/22224444 e/jamesho@gmail.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+Brings back the expired tasks/events back to active list, with the deadline/event date altered.The index is as shown by `listc` command.
 
-* **Clear** : `clear`
+Format : `Recycle INDEX DEADLINE | START_DATE START_TIME to [END_DATE] END_TIME`
 
-* **Delete** : `delete INDEX` <br>
-   e.g. `delete 3`
+### 3.13 Files
 
-* **Find** : `find KEYWORD [MORE_KEYWORDS]` <br>
-  e.g. `find James Jake`
+**Switch between files**: `switchf FILEPATH`
+> if the file given by FILEPATH does not exist, `TaskCrusher` creates a new file
 
-* **List** : `list` <br>
-  e.g.
+## 4. FAQ
 
-* **Help** : `help` <br>
-  e.g.
+**Q**: How do I create my own storage file to import into the app?<br>
+**A**: Refer to [Storage file format](#storage-file-format) for instructions
 
-* **Select** : `select INDEX` <br>
-  e.g.`select 2`
+**Q**: How do I import my own storage file into the app?<br>
+**A**: Back up your current storage file as appropriate and overwrite it with the storage file you wish to use.
+
+**Q**: How do I transfer my data to another Computer?<br>
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous TaskCrusher installation.
 
+## 6. Storage File Format
```

Comment:
seems incomplete? if this is not relevant to you, you can remove it from user guide.